src/ore/src/test.rs

```diff
@@ -42,7 +42,7 @@ pub fn init_logging() {
 /// not run tests in any particular order, each must call `init_logging`.
 pub fn init_logging_default(level: &str) {
     LOG_INIT.call_once(|| {
-        let filter = EnvFilter::try_from_env("MZ_LOG_FILTER")
+        let filter = EnvFilter::try_from_env("MZ_STARTUP_LOG_FILTER")
```

Comment:
Hm, I feel like this is weird, because it’s not like this is the “startup” log filter for the test. Maybe this should use a new bar like MZ_TEST_LOG_FILTER?

Reply:
FWIW my vote is we go back to having a single `MZ_LOG_FILTER` env variable that also gets plumbed to `clusterd`.

Reply:
Ah, that was intentionally removed so that the log filter could be controlled via LaunchDarkly!

Reply:
Changed and added a comment in the testing guide. RFAL.